src/main/java/frc/robot/subsystems/climber/commands/SetGearbox.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.climber.Climber;
+
+public class SetGearbox extends InstantCommand {
+    private final Climber climber;
+    private Climber.PistonMode mode;
+
+    public SetGearbox(Climber climber, Climber.PistonMode mode) {

```

Comment:
1) I think `ShiftGear` is a better and more concise name. 
2) Be consistent. You can't call the the command `SetGearbox`, but supply it a `PistonMode`. Isn't a boolean enough?